app/src/ui/business-card/BusinessCard.module.scss

```diff
@@ -0,0 +1,209 @@
+@import "src/theme/base";
+
+$card-border: #e6ebf0;
+$card-background: #ffff;
+$card-subtitle: #898f99;
+
+$card-width: 253px;
+$card-height: 295px;
+$card-default-bottom: 0.2rem;
+
+@mixin withContent {
+  position: relative;
+  top: 120px;
+  height: 500px;
+  display: flex;
+  flex: 1 1 auto;
+  flex-direction: column;
+  padding-right: 1rem;
+  padding-left: 1rem;
+  background: $card-background;
```

Comment:
Color vars are set in `global.scss` as CSS variables. This allow us to change the Theme from Dark to Light, etc.